### PR TITLE
2.8.0

### PR DESCRIPTION
**Pull request by Mixa84**
#### What kind of PR is this?:

/kind chore

#### What this PR does / why we need it:
Setting version and heights for v2.8.0

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -477,6 +477,7 @@ void SetupServerArgs()
     gArgs.AddArg("-fortcanningparkheight", "Fort Canning Park fork activation height (regtest only)", ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::CHAINPARAMS);
     gArgs.AddArg("-fortcanninghillheight", "Fort Canning Hill fork activation height (regtest only)", ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::CHAINPARAMS);
     gArgs.AddArg("-fortcanningroadheight", "Fort Canning Road fork activation height (regtest only)", ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::CHAINPARAMS);
+    gArgs.AddArg("-fortcanningspicegardenheight", "Fort Canning Split fork activation height (regtest only)", ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::CHAINPARAMS);
```
Comment (Jouzo) [inline src/init.cpp:480]:
```suggestion
    gArgs.AddArg("-fortcanningspicegardenheight", "Fort Canning Spice Garden fork activation height (regtest only)", ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::CHAINPARAMS);
```

**Review by Jouzo — DISMISSED** [review]

File: src/miner.cpp [hunk 2]
```diff
@@ -708,19 +708,19 @@ namespace pos {
...
+    // in order to give time for subnode 0 to first succeed and create a record.
+    //
     // This is done to ensure that we can workaround the bug, without waiting for consensus change in next update.
     // This shouldn't have any effect on the mining, even though we ignore some successful hashes since the
-    // Coinages are retained and keep growing. Once subnode 0 mines, subnode 1 should stake again shortly with 
+    // Coinages are retained and keep growing. Once subnode 0 mines, subnode 1 should stake again shortly with
     // higher coinage / TM.
     //
-    bool shouldIgnoreMint(uint8_t subNode, int64_t blockHeight, int64_t creationHeight, 
+    bool shouldIgnoreMint(uint8_t subNode, int64_t blockHeight, int64_t creationHeight,
```
Comment (prasannavl) [inline src/miner.cpp:721]: We can remove this entirely, actually.

File: src/miner.cpp [hunk 3]
```diff
@@ -828,7 +828,7 @@ namespace pos {
                     if (pos::CheckKernelHash(stakeModifier, nBits, creationHeight, blockTime, blockHeight, masternodeID, chainparams.GetConsensus(),
                                              subNodesBlockTime, timelock, ctxState))
                     {
-                        if (shouldIgnoreMint(ctxState.subNode, blockHeight, creationHeight, subNodesBlockTime, chainparams)) 
+                        if (shouldIgnoreMint(ctxState.subNode, blockHeight, creationHeight, subNodesBlockTime, chainparams))
```
Comment (prasannavl) [inline src/miner.cpp:831]: And this too.

File: src/miner.cpp [hunk 4]
```diff
@@ -853,7 +853,7 @@ namespace pos {
...
```
Comment (prasannavl) [inline src/miner.cpp:856]: And this

**Review by prasannavl — COMMENTED** [review]
Minor ones, unrelated, but could use the opportunity to clean up

**Review by Bushstar — APPROVED** [review]

**Pull request merged**